pyvo/dal/tests/test_adhoc.py

```diff
@@ -62,7 +62,7 @@ class TestClass(dict, AxisParamMixin):
 
     # errors
     test_obj.pos.pop()
-    with pytest.raises(ValueError):
```

Comment:
@andamian - we haven't documented the exception type for this, but changing these ValueErrors to TypeErrors are technically an API change, even though it was undocumented API.

Would you be OK with going ahead with it, or should I hack back ValueErrors instead?